tests/mocha/variable_map_test.js

```diff
@@ -246,6 +246,54 @@ suite('Variable Map', function () {
     });
   });
 
+  suite('changeVariableType', function () {
+    test('normally', function () {
```

Comment:
I am slightly picky about test names :P Can you change these to use the format "tests that ___"? E.g. `test('changing the type of a variable affects getType and getVariablesOfType')`, or something like that?

Might also make sense to split this into two tests, since right now your tests are asserting for `getType` and for `getVariablesOfType`

Reply:
Updated the test description; as discussed offline, `getType` and `getVariablesOfType` are exercised separately elsewhere, and are just being used here in order to verify that `changeVariableType` did so.